ribbon-archaius/src/main/java/com/netflix/client/config/DefaultClientConfigImpl.java

```diff
@@ -344,89 +347,105 @@ public int getDefaultMaxTotalHttpConnections() {
         return DEFAULT_MAX_TOTAL_HTTP_CONNECTIONS;
     }
 
+    @Deprecated
     public int getDefaultMaxConnectionsPerHost() {
         return DEFAULT_MAX_CONNECTIONS_PER_HOST;
     }
 
+    @Deprecated
     public int getDefaultMaxTotalConnections() {
         return DEFAULT_MAX_TOTAL_CONNECTIONS;
     }
 
+    @Deprecated
     public float getDefaultMinPrimeConnectionsRatio() {
         return DEFAULT_MIN_PRIME_CONNECTIONS_RATIO;
     }
 
+    @Deprecated
     public String getDefaultPrimeConnectionsClass() {
         return DEFAULT_PRIME_CONNECTIONS_CLASS;
     }
 
+    @Deprecated
     public String getDefaultSeverListClass() {
         return DEFAULT_SEVER_LIST_CLASS;
     }
 
+    @Deprecated
     public int getDefaultConnectionIdleTimertaskRepeatInMsecs() {
         return DEFAULT_CONNECTION_IDLE_TIMERTASK_REPEAT_IN_MSECS;
     }
 
+    @Deprecated
     public int getDefaultConnectionidleTimeInMsecs() {
         return DEFAULT_CONNECTIONIDLE_TIME_IN_MSECS;
     }
 
-    public VipAddressResolver getResolver() {
```

Comment:
Blanket question for all the other method removals, but you confirmed with code search that these aren't in use?

Reply:
This is one just got moved down because it's position in the class was causing my OCD to flare up